大一/数据结构/习题1/test1.cpp

```diff
@@ -0,0 +1,20 @@
+﻿#include <iostream>
+using namespace std;
+
+void hanoi(int n, char a, char c, char b) {
```

Comment:
1. Usually we put the last `{` to next new line like this:
```
void hanoi(int n, char a, char c, char b)
{
    what_ever_left_in_ur_code;
}
```

2. Missing description about each var, pls rename n, a, c, b to more meaningful name and add proper comment to them.